### DISCUSSION TITLE
Reference to object.

### DISCUSSION THREAD

**Discussion by Devjughead** [serenity-is/Serenity#6429] [Help]
Reference to object not set as an instance of an object, the error appears when I want to save in a table from another form with lookup.

![IMG-20220627-WA0038](https://user-images.githubusercontent.com/108319630/176096923-b5e40dee-0b36-4a77-a406-c2671310a0ec.jpeg)

**Comment by edwardch** [comment]
You should put a breakpoint in your save handler step through your code and watch the properties. 

Good luck!

**Comment by stixoffire** [comment]
The error means an object does not exist as an instance (it might not be instantiated). The name might not match CASE MATTERS.
MyDoggieForm is different than myDoggieForm or mydoggieform.. 
Also you are using a lookup - did you use lookup include ? In your parent row .. (assuming you have a parent child relationship.

Very difficult to know what you need since you did not post any of your code! 
Not even a functional flow of your code ..!!

My ESP is really not that good .. so for now all I can offer is work it out - because that error means exactly what it says - and it is a standard c# coding error message.
